src/app-logic/url-handling.js

```diff
@@ -348,7 +348,9 @@ export function getQueryStringFromUrlState(urlState: UrlState): string {
       stringifyCommittedRanges(urlState.profileSpecific.committedRanges) ||
       undefined,
     thread:
-      selectedThreads === null ? undefined : [...selectedThreads].join(','),
+      selectedThreads === null
+        ? undefined
+        : encodeUintSetForUrlComponent(selectedThreads),
```

Comment:
That's true but I've never done that. Have you?

Reply:
Occasionnaly, but very rarely, I think mostly for testing. I think this could be handy as a result of a script, to autoselect some thread.. But this is still possible, and somebody could even target an old url version if necessary, or we could provide a "real API" for this if this is really needed.